sentinel-core/src/main/java/com/alibaba/csp/sentinel/Entry.java

```diff
@@ -66,6 +71,31 @@
     private Throwable error;
     private BlockException blockError;
 
+
+    private HashMap<Long,Timeout> tokenInfoMap;
```

Comment:
Adding too many new attributes in `Entry` may not be a good idea. If really needed, we may unify it as a map of metadata.

Reply:
If the attributes  is not necessary, this map will not be initialized